parser.y

```diff
@@ -5764,11 +5764,26 @@ AdminStmt:
 			Index: string($5),
 		}
 	}
-|	"ADMIN" "RESTORE" "TABLE" "BY" "JOB" NumList
+|	"ADMIN" "RESTORE" "TABLE" "BY" "JOB" NUM
 	{
 		$$ = &ast.AdminStmt{
 			Tp: ast.AdminRestoreTable,
-			JobIDs: $6.([]int64),
+			JobIDs: []int64{$6.(int64)},
+		}
+	}
+|	"ADMIN" "RESTORE" "TABLE" TableName
+	{
+		$$ = &ast.AdminStmt{
+			Tp: ast.AdminRestoreTable,
+			Tables: []*ast.TableName{$4.(*ast.TableName)},
+		}
+	}
+|	"ADMIN" "RESTORE" "TABLE" TableName NUM
```

Comment:
What does `NUM` mean? I think if we want use job number, we can use line5767.

Reply:
@zimulala 
`admin restore table [table_name] [NUM]`, NUM means search how many recent ddl history jobs. Such as `admin restore table t1 20`, we will only search the recent 20 history ddl jobs to find the job which drop table t1.

Reply:
But this check has not yet implemented in tidb. Because we currently always get all ddl history job, after implement only get [NUM] ddl history jobs, I will implement this in TiDB.